web/src/lib/static/loadMunicipalities.test.ts

```diff
@@ -1,8 +1,9 @@
 import { generateMunicipalityDetail } from "@businessnjgovnavigator/shared";
-import fs from "fs";
+import fs from "fs"; // Object
 import { loadAllMunicipalities } from "./loadMunicipalities";
 
-jest.mock("fs");
+jest.mock("fs"); // Object with Jest Fns
```

Comment:
We are importing `fs` and using `jest.mock("fs");` in multiple places throughout our codebase, why comment these now?

[pebble] remove commented code

Reply:
Good catch, these comments were used to further explain the concepts of mocking, I didn't intend them to be there